hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/BaseFreonGenerator.java

```diff
@@ -92,6 +98,18 @@ public class BaseFreonGenerator {
       defaultValue = "10")
   private int threadNo;
 
+  @Option(names = {"--timebase"},
+      description = "If set, freon will run for the duration of the --runtime"
+          + " specified even if the --number-of-tests operation"
+          + " has been completed.",
+      defaultValue = "false")
+  private boolean timebase;
+
+  @Option(names = {"--runtime"},
+      description = "Tell freon to terminate processing after"
+          + "the specified period of time in seconds.")
+  private long runtime;
```

Comment:
I'm not sure how --min-duration and --max-duration relate to -n.
such as the command `freon xxx --min-duration 100  --max-duration 1000 -n 1` how long the test will be running?
Can you give me some examples or references?

Reply:
`--min-duration` is like `--timebase --runtime`, while `--max-duration` is like `--runtime`.

`--min-duration m --max-duration M -n N` runs for at least `m` seconds.  If `>= N` items are done, it stops.  If not, it continues until reaching either item count or `--max-duration` time limit.

Reply:
yeah, I understand 
if the `-n N --min-duration m` is given, the test will run `m` seconds even though it has been executed `N` times.
if the `-n N --max-duration M` is given, the test will run `M` seconds even though it has been not executed `N` times or executed `N` times if the M > "Time spent executing `N` times"
Tell me if I'm wrong.

Reply:
For the `--timebase` and  `--runtime`

specify a runtime: 
- `-n N --timebase --runtime M`, the test will run `M` seconds no matter how many times are executed.

specify run times:
- `-n N`, test will run `N` times

the `--timebase --runtime M` All must be given, only given `--runtime M` is not supported.

I think this may be easy to understand

Reply:
@errose28 The option of `--min-duration and --max-duration` or `--timebase and --runtime` which do you think is better. If easy to understand is more important, I think `--timebase and --runtime` is easier to understand (But maybe I have some issues that I haven't considered).

Reply:
@adoroszlai PTAL

Reply:
@xichen01 I have suggested `--min-duration` and `--max-duration` because I think these are easier to understand.

I have the following issues with the proposed options:
 * `--runtime` has different meaning depending on another option (`--timebase`)
 * `--timebase` is invalid without `--runtime`
 * both names are cryptic

Reply:
Ok, I will change it to the suggested "--min-duration" and "--max-duration".
But if both `-n N` and `--min/max-duration X` are given the real-time progress bar may be difficult to implement, because the final run time is uncertain. So in this case, instead of printing the real-time progress bar, the number of executions and the executions time will be printed
such as:
`operations executed times: 100, runtime: 30s`
if the "--min-duration" and "--max-duration" are both not given, the progress bar will the same as before

Reply:
I removed the `timebase` option and renamed the `runtime` option to `duration`.
for now:
- `freon -n N` the test will execute N times.
- `freon --duration T ` the test  will execute the duration of T

@adoroszlai Do you think this is an acceptable implementation?

Reply:
> I removed the `timebase` option and renamed the `runtime` option to `duration`.

Thanks @xichen01, that sounds good.